cdap-ui/app/hydrator/templates/partial/node-config-modal/node-config/plugin-edit-output-schema.html

```diff
@@ -69,7 +69,7 @@ <h4>
       node="HydratorPlusPlusNodeConfigCtrl.state.node"
       groups-config="HydratorPlusPlusNodeConfigCtrl.state.groupsConfig"
       update-default-output-schema="HydratorPlusPlusNodeConfigCtrl.updateDefaultOutputSchema(outputSchema)"
-      is-disabled="isDisabled">
+      is-disabled="isDisabled || HydratorPlusPlusNodeConfigCtrl.datasetAlreadyExists">
```

Comment:
Would this be a problem if the dataset already exist but doesn't have a schema available from backend?

Reply:
No you can still edit the schema in that case